controller/src/main/java/io/pravega/controller/server/impl/ControllerServiceConfigImpl.java

```diff
@@ -101,6 +95,7 @@
         this.restServerConfig = restServerConfig;
         this.retentionFrequency = retentionFrequency == null ? Duration.ofMinutes(Config.MINIMUM_RETENTION_FREQUENCY_IN_MINUTES)
                 : retentionFrequency;
+        this.healthCheckFrequency = healthCheckFrequency == null ? Duration.ofSeconds(10) : healthCheckFrequency;
```

Comment:
Is there a way we can default this the same way it is done above (see `this.retentionFrequency`). We'd have to add a field to `Config` for that.